INSTALL.md

```diff
@@ -125,6 +125,85 @@ source.
 
 ### Windows ###
 
+Windows installation support has recently been enhanced. The new installation
+method doesn't require Cygwin or WSL and relies only a light weight git/bash
+distribution like Git-Bash, and the Intel OneAPI base distribution as well as
+the Intel OneAPI HPC Toolkit. This allows native and redistributable statically
+linked binaries to be built if you use the appropriate compiler flags. Clients
+will only require the free redistributable Intel MPI runtime.
+
+#### New and Recommended Windows Installation ####
+
+Please follow these instructions, this is the recommended way to install
+OpenCoarrays on Windows.
+
+##### Pre-requisits #####
+
+* [CMake]
+* [Git-Bash]
+* [GCC, GFortran, Unix Make]
+* [Intel OneAPI Base] and [Intel OneAPI HPC toolkit]
+
+##### Installation Procedure #####
+
+1. Ensure all the prerequisites have been installed following the directions of
+   the provider.
+2. Launch the 64-bit Intel OneAPI developer shell. This will either be in the
+   start menu, or you may need to figure out how to launch it manually. See the
+   note below for help.
+3. Ensure that cmake, git-bash, gfortran, gcc, make are available on your default
+   path. The installers should have taken care of this for you.
+5. Launch a Git-Bash bash shell from the OneAPI environment, or ensure Git-Bash's
```

Comment:
This sentence describes *both* of the substeps--substep 1 corresponds to "Launch a Git-Bash bash shell from the OneAPI environment" and substep 2 corresponds to "ensure Git-Bash's `bash.exe` is first on your path". Therefore I think the original text makes more sense unless you think there is a problem with my approach/description or can think of a clearer way to phrase it.

> If I launch a bash shell, it no longer has access to `mpifc`. It still has access to `ifort`, so I'm not sure why.

This is because of the way Windows handles file types: by filename extension and because the Intel compiler wrapper scripts are batchfile scripts. When you call `mpifc` `cmd.exe` (or powershell))